geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/QueryPlanner.scala

```diff
@@ -215,7 +213,7 @@ object QueryPlanner extends LazyLogging {
   }
 
   def setPerThreadQueryHints(hints: Map[AnyRef, AnyRef]): Unit = threadedHints.put(hints)
-  def clearPerThreadQueryHints() = threadedHints.clear()
+  def clearPerThreadQueryHints(): Unit = threadedHints.clear()
```

Comment:
haha, intellij has started flagging public methods without explicit return types, so I've started filling them in :P